intra/relay/templates/deployment.yaml

```diff
@@ -0,0 +1,26 @@
+{{ range $i, $stream := .Values.streams }}
+---
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: relay-{{ $stream }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: relay-{{ $stream }}
+  template:
+    metadata:
+      labels:
+        app: relay-{{ $stream }}
+    spec:
+      containers:
+      - name: relay
+        image: paulbouwer/hello-kubernetes:1.5
+        ports:
+        - containerPort: 8080
+        env:
+          - name: MESSAGE
+            value: {{ $stream }}
```

Comment:
Could perhaps be replaced with `args: ["{{ $stream }}"]` if docker image utilises an `ENDPOINT`.

Reply:
The Docker file in ntrip-mqtt-relay uses entry point. You would provide `args: [ "-caster", "http://caster:2101/{{ $stream }}", "-broker", "tcp://vernemq:1883", "-topic", "'{{ $stream }}'", "-username", "username", "-password", "pass"]`